Timesheet.gs

```diff
@@ -39,18 +39,25 @@ var Log_
 
 var EVENT_HANDLERS = {
 
-//                         Initial actions  Name                         onError Message                        Main Functionality
-//                         ---------------  ----                         ---------------                        ------------------
-
-  checkIn:                 [function() {},  'checkIn()',                'checkIn failed',                       checkIn_],
-  checkOut:                [function() {},  'checkOut()',               'checkOut failed',                      checkOut_],
+//                         Name                         onError Message                        Main Functionality
+//                         ----                         ---------------                        ------------------
+  onInstall:               ['onInstall()',              'Failed to install',                    onInstall_],
+  initialize:              ['initialize()',             'Failed to initialize',                 initialize_],
+  checkIn:                 ['checkIn()',                'checkIn failed',                       checkIn_],
+  checkOut:                ['checkOut()',               'checkOut failed',                      checkOut_],
+  isCheckedIn:             ['isCheckedIn()',            'isCheckedIn failed',                   isCheckedIn_],
```

Comment:
For a moment I wondered where getStatus() had gone :)

Reply:
;-)